hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/OperationExecutor.java

```diff
@@ -154,11 +154,6 @@
     boolean isOperationThread();
 
     /**
-     * Interrupts the partition threads.
-     */
-    void interruptPartitionThreads();
```

Comment:
This method was introduced specifically to support Hot Restart. Since Hot Restart doesn't call it anymore, I find it best to delet.